lib/core/buildFullPath.js

```diff
@@ -15,7 +15,7 @@ import combineURLs from '../helpers/combineURLs.js';
  */
 export default function buildFullPath(baseURL, requestedURL, allowAbsoluteUrls) {
   let isRelativeUrl = !isAbsoluteURL(requestedURL);
-  if (baseURL && isRelativeUrl || allowAbsoluteUrls == false) {
+  if (baseURL && (isRelativeUrl || allowAbsoluteUrls == false)) {
```

Comment:
i noticed that the docstring for buildFullPath is missing an entry for the `allowAbsoluteUrls` param on line 13 above. this would be a great opportunity to add this in

Reply:
feel free to do that in another pr